roles/nginx/templates/alfresco.conf.j2

```diff
@@ -16,7 +16,7 @@ upstream repo_lb {
 
 upstream tracker_lb {
 {% for repo in repo_hosts | unique %}
-        server {{ repo.local_addr | default(repo.inventory_name) }}:{{ ports_cfg.repository.http | default(8080) }} max_fails=2 fail_timeout=8s {% if not repo.cluster_keepoff | default(false) and repo_hosts | selectattr('cluster_keepoff', 'true') %}backup{% endif %}; 
+        server {{ repo.local_addr | default(repo.inventory_name) }}:{{ ports_cfg.repository.http | default(8080) }} max_fails=2 fail_timeout=8s {% if not repo.cluster_keepoff | default(false) and repo_hosts | selectattr('cluster_keepoff') %}backup{% endif %};
```

Comment:
I've done further tests and it looks like this implicit boolean test actually evaulate every non empty string to `true`(includeing "false & "null"). This can be very missleading and we should keep the previous test from for `selectattr()`